### PR TITLE
Ruletest 23-7 Test JSON

### PR DESCRIPTION
**Pull request by jugonzal07**
First cut at 23-7 ruletest JSON

### CODE REVIEW

**Review by yunjoonjung-PNNL — COMMENTED** [review]
Same as PR #891. I'd appreciate it if you could include ```fan_configuration```.

Other than this, everything looks great to me.

**Comment by jugonzal07** [comment]
> Same as PR #891. I'd appreciate it if you could include `fan_configuration`.
> 
> Other than this, everything looks great to me.

Addressed.

File: rct229/ruletest_engine/ruletest_jsons/section23/rule_23_7.json [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+                                                "id": "Thermal Zone 1",
+                                                "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                                "terminals": [
+                                                    {
+                                                        "id": "VAV Air Terminal 1",
+                                                        "is_supply_ducted": true,
+                                                        "type": "VARIABLE_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "System 6",
+                                                        "heating_source": "ELECTRIC"
```
Comment (yunjoonjung-PNNL) [inline rct229/ruletest_engine/ruletest_jsons/section23/rule_23_7.json:43]: Juan, could you also add '''fan_configuration''' here?

Comment (jugonzal07) [reply rct229/ruletest_engine/ruletest_jsons/section23/rule_23_7.json]: Thanks Yun Joon, I missed one of the terminals. Updated!

**Comment by weilixu** [comment]
@jugonzal07 @yunjoonjung-PNNL is this PR ready to close?

**Comment by yunjoonjung-PNNL** [comment]
> @jugonzal07 @yunjoonjung-PNNL does this PR ready to close?

@weilixu  This PR can be closed because the RMD in this PR is the same as the one in the develop branch.

**Comment by jugonzal07** [comment]
Thank you Yun Joon! No reason for it if it's already in the development branch.

**Pull request closed without merging**